tutorials/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Tutorials
+suppressRelated: true
+---
+
+These tutorials will give you step-by-step instruction to help you learn NServiceBus.
+
+### [NServiceBus 101: Messaging Basics](nservicebus-101/)
+
+In this course, you will learn how to get working quickly with NServiceBus, including how to set up your development environment and create messaging endpoints, defining messages and handlers, how to send messages and publish events, and how to deal with exceptions and take advantage of automatic retries.
+
+### Additional courses
+
+Please [sign up for updates](https://particular.net/s/new-course-signup) if you'd like to be notified when we release additional courses.
```

Comment:
we already have a feedback mechanism via either contact us or the edit on GH button.

We want out interactions with this content to be handled in a consistent manner. ATM that manner is GH.

We already capture metrics via google analytics

if we wanted them to be notified of new updates to content we should do that globally for all documents. this content is not a special case.

Basically nothing about this content means it should be handled differently in regards to feedback, notification of updates, analytics or requesting new updates.